### PR TITLE
Small setup doc for the app store option

### PR DESCRIPTION
**Pull request by xster**

### CODE REVIEW

**Review by cbracken — APPROVED** [review]

**Comment by cbracken** [comment]
lgtm

File: packages/flutter_tools/lib/src/ios/ios_workflow.dart [hunk 1]
```diff
@@ -132,6 +132,7 @@ class IOSWorkflow extends DoctorValidator implements Workflow {
         messages.add(new ValidationMessage.error(
             'Xcode installation is incomplete; a full installation is necessary for iOS development.\n'
             'Download at: https://developer.apple.com/xcode/download/\n'
+            'Or install Xcode via the App Store\n'
```
Comment (Hixie) [inline packages/flutter_tools/lib/src/ios/ios_workflow.dart]: nit: period at the end of the sentence.

File: packages/flutter_tools/lib/src/ios/ios_workflow.dart [hunk 2]
```diff
@@ -132,6 +132,7 @@ class IOSWorkflow extends DoctorValidator implements Workflow {
...
             'Once installed, run \'sudo xcode-select --switch /Applications/Xcode.app/Contents/Developer\'.'
```
Comment (Hixie) [inline packages/flutter_tools/lib/src/ios/ios_workflow.dart]: while you're here, can you change this to:
`'Once installed, run: "sudo xcode-select --switch /Applications/Xcode.app/Contents/Developer"'`

File: packages/flutter_tools/lib/src/ios/ios_workflow.dart (hunk 1 above)
Comment (xster) [inline packages/flutter_tools/lib/src/ios/ios_workflow.dart]: done

File: packages/flutter_tools/lib/src/ios/ios_workflow.dart (hunk 2 above)
Comment (xster) [inline packages/flutter_tools/lib/src/ios/ios_workflow.dart]: Just formatted it like the rest of the file with an indent

**Pull request merged**